app/dashboard/views.py

```diff
@@ -802,51 +804,45 @@ def users_fetch(request):
         )
 
     if rating != 0:
-        user_list = user_list.annotate(
+        profile_list = profile_list.annotate(
             average_rating=Avg('feedbacks_got__rating', filter=Q(feedbacks_got__bounty__network=network))
         ).filter(
             average_rating__gte=rating
         )
 
     if organisation:
-        user_list = user_list.filter(
+        profile_list = profile_list.filter(
             fulfilled__bounty__network=network,
             fulfilled__bounty__accepted=True,
             fulfilled__bounty__github_url__icontains=organisation
         ).distinct()
 
     params = dict()
-    all_pages = Paginator(user_list, limit)
+    all_pages = Paginator(profile_list, limit)
     all_users = []
     for user in all_pages.page(page):
         profile_json = {}
-        profile_json = user.to_standard_dict()
-        if user.avatar_baseavatar_related.exists():
-            user_avatar = user.avatar_baseavatar_related.first()
-            profile_json['avatar_id'] = user_avatar.pk
-            profile_json['avatar_url'] = user_avatar.avatar_url
         count_work_completed = Activity.objects.filter(profile=user, activity_type='work_done').count()
         count_work_in_progress = Activity.objects.filter(profile=user, activity_type='start_work').count()
-        previously_worked_with = 0
-        if profile:
-            previously_worked_with = BountyFulfillment.objects.filter(
-                bounty__bounty_owner_github_username__iexact=profile.handle,
-                fulfiller_github_username__iexact=user.handle,
-                bounty__network=network,
-                bounty__accepted=True
-            ).count()
-
+        profile_json = {k: getattr(user, k) for k in
```

Comment:
slick! Style-wise, lets try to align with PEP8 style - https://www.python.org/dev/peps/pep-0008/
```
profile_json = {
    k: getattr(user, k) for k in
    ['id', 'actions_count', 'created_on', 'handle', 'hide_profile',
     'show_job_status', 'job_location', 'job_salary', 'job_search_status',
     'job_type', 'linkedin_url', 'resume', 'remote', 'keywords',
     'organizations', 'is_org']}
```

You can check with http://pep8online.com/ - I want to integrate a python stylechecker into our makefile/tests as well